security/pkg/pki/ca/controller/monitoring.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	configID = "SecretController"
```

Comment:
any reason to use mixed case here?

and is this needed?  you already use `secret_controller` as the subsystem.

Reply:
Removed.